TwitterPolling/poll-trends.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+#-----------------------------------------------------------------------
+# twitter-trends
+#  - lists the current global trending topics
+#-----------------------------------------------------------------------
+
+from twitter import *
+
+#-----------------------------------------------------------------------
+# load our API credentials
+#-----------------------------------------------------------------------
+import sys
+sys.path.append(".")
+import config
+
+#-----------------------------------------------------------------------
+# create twitter API object
+#-----------------------------------------------------------------------
+twitter = Twitter(auth = OAuth(config.access_key,
+                  config.access_secret,
+                  config.consumer_key,
+                  config.consumer_secret))
+
+#-----------------------------------------------------------------------
+# retrieve global trends.
+# other localised trends can be specified by looking up WOE IDs:
+#   http://developer.yahoo.com/geo/geoplanet/
+# twitter API docs: https://dev.twitter.com/rest/reference/get/trends/place
+#-----------------------------------------------------------------------
+ny_results = twitter.trends.place(_id = 2459115)
+
+la_results = twitter.trends.place(_id = 2442047)
+
+chicago_results = twitter.trends.place(_id = 2379574)
+
+dallas_results = twitter.trends.place(_id = 2388929)
+
+dc_results = twitter.trends.place(_id = 2514815)
+
+all_results = ny_results+la_results+chicago_results+dallas_results+dc_results
```

Comment:
should probably autoformat, so there are some spaces

Reply:
not sure how, just rolling with Sublime for rn